packages/kbn-plugin-generator/sao_template/template/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "<%= kebabCase(name) %>",
+  "version": "0.0.0",
+  "description": "<%= description %>",
+  "main": "index.js",
+  "kibana": {
+    "version": "<%= (kbnVersion === 'master') ? 'kibana' : kbnVersion %>",
+    "templateVersion": "<%= templateVersion %>"
+  },
+  "scripts": {
+    "kbn": "node ../../kibana/scripts/kbn",
```

Comment:
What do you think about also including this `preinstall` script: https://github.com/elastic/x-pack-kibana/blob/master/package.json#L6? (https://github.com/elastic/kibana/blob/master/preinstall_check.js)